packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/billing-tile/BillingTile.component.tsx

```diff
@@ -24,19 +33,29 @@ export default function BillingTile({ id }: TGetVcdServiceIdParams) {
         <Description>{user?.email}</Description>
         <OsdsDivider separator />
         <TileSubtitle>{t('managed_vcd_dashboard_service_renew')}</TileSubtitle>
-        <Links
-          onClickReturn={() => {}}
-          label={dateTimeFormat?.format(
-            // new Date(billingService.data.billing.nextBillingDate),
-            new Date(),
-          )}
-        />
+        {nextBillingDate ? (
+          <Links
+            onClickReturn={() => {}}
+            label={dateTimeFormat?.format(new Date(nextBillingDate))}
+          />
+        ) : (
+          <span>-</span>
+        )}
         <OsdsDivider separator />
         <TileSubtitle>{t('managed_vcd_dashboard_password')}</TileSubtitle>
-        <Links
-          onClickReturn={() => {}}
-          label={t('managed_vcd_dashboard_password_renew')}
-        />
+        <div className="flex-wrap">
+          <OsdsLink disabled>
```

Comment:
Can't we use links instead of OsdsLink here ? is it because of the disabled props ?

Reply:
yes, otherwise link can't be disabled